README.md

```diff
@@ -128,37 +129,38 @@ This type system has practical significance. For example, when computing pairwis
-|  WeightedEuclidean   |  `weuclidean(x, y, w)`     | `sqrt(sum((x - y).^2 .* w))`  |
-|  WeightedSqEuclidean |  `wsqeuclidean(x, y, w)`   | `sum((x - y).^2 .* w)`  |
-|  WeightedCityblock   |  `wcityblock(x, y, w)`     | `sum(abs(x - y) .* w)`  |
-|  WeightedMinkowski   |  `wminkowski(x, y, w, p)`  | `sum(abs(x - y).^p .* w) ^ (1/p)` |
-|  WeightedHamming     |  `whamming(x, y, w)`       | `sum((x .!= y) .* w)`  |
+| type name                 |  convenient syntax         | math definition     |
+| --------------------      | -------------------------- | --------------------|
+|  Euclidean                |  `euclidean(x, y)`         | `sqrt(sum((x - y) .^ 2))` |
+|  SqEuclidean              |  `sqeuclidean(x, y)`       | `sum((x - y).^2)` |
+|  Cityblock                |  `cityblock(x, y)`         | `sum(abs(x - y))` |
+|  Chebyshev                |  `chebyshev(x, y)`         | `max(abs(x - y))` |
+|  Minkowski                |  `minkowski(x, y, p)`      | `sum(abs(x - y).^p) ^ (1/p)` |
+|  Hamming                  |  `hamming(k, l)`           | `sum(k .!= l)` |
+|  Rogers-Tanimoto          |  `rogerstanimoto(a, b)`    | `2(sum(a&!b) + sum(!a&b)) / (2(sum(a&!b) + sum(!a&b)) + sum(a&b) + sum(!a&!b))` |
+|  Jaccard                  |  `jaccard(x, y)`           | `1 - sum(min(x, y)) / sum(max(x, y))` |
+|  CosineDist               |  `cosine_dist(x, y)`       | `1 - dot(x, y) / (norm(x) * norm(y))` |
+|  CorrDist                 |  `corr_dist(x, y)`         | `cosine_dist(x - mean(x), y - mean(y))` |
+|  ChiSqDist                |  `chisq_dist(x, y)`        | `sum((x - y).^2 / (x + y))` |
+|  KLDivergence             |  `kl_divergence(p, q)`     | `sum(p .* log(p ./ q))` |
+|  GenKLDivergence          |  `gkl_divergence(x, y)`    | `sum(p .* log(p ./ q) - p + q)` |
+|  RenyiDivergence          | `renyi_divergence(p, q, k)`| `log(sum( p .* (p ./ q) .^ (k - 1))) / (k - 1)` |
+|  JSDivergence             |  `js_divergence(p, q)`     | `KL(p, m) / 2 + KL(p, m) / 2 with m = (p + q) / 2` |
+|  SpanNormDist             |  `spannorm_dist(x, y)`     | `max(x - y) - min(x - y )` |
+|  BhattacharyyaDist        |  `bhattacharyya(x, y)`     | `-log(sum(sqrt(x .* y) / sqrt(sum(x) * sum(y)))` |
+|  HellingerDist            |  `hellinger(x, y) `        | `sqrt(1 - sum(sqrt(x .* y) / sqrt(sum(x) * sum(y))))` |
+|  Mahalanobis              |  `mahalanobis(x, y, Q)`    | `sqrt((x - y)' * Q * (x - y))` |
+|  SqMahalanobis            |  `sqmahalanobis(x, y, Q)`  | ` (x - y)' * Q * (x - y)`  |
+|  MeanAbsDeviation         |  `meanad(x, y)`            | `mean(abs.(x - y))` |
+|  MeanSqDeviation          |  `msd(x, y)`               | `mean(abs2.(x - y))` |
+|  RMSDeviation             |  `rmsd(x, y)`              | `sqrt(msd(x, y))` |
+|  NormRMSDeviation         |  `nrmsd(x, y)`             | `rmsd(x, y) / (maximum(x) - minimum(x))` |
+| Bray-Curtis Dissimliarity |  `braycurtis(x, y)`        | `sum(abs(x - y)) / sum(abs(x+y))`  |
```

Comment:
IIUC, it should be just BrayCutris, since in this column you provide the type names to be used from e.g. `pairwise()` method.
So increasing the column width would be also unnecessary.

Reply:
@alyst Sorry for delayed response (holiday). I'll take a look at this today.

Reply:
Fixed.